oap-server/server-storage-plugin/storage-jdbc-hikaricp-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/SQLExecutor.java

```diff
@@ -38,6 +40,8 @@ public class SQLExecutor implements InsertRequest, UpdateRequest {
 
     private String sql;
     private List<Object> param;
+    @Getter
+    private final List<SQLExecutor> additionalSQLs = new ArrayList<>();
```

Comment:
It is better to do initialization in `#appendAdditionalSQLs`, as in most SQLExecutors, there is no `additionalSQLs`, you could reduce GC/Memory-allocate cost from this.